### PR TITLE
[IMP] website: 404 page improvement

### PR DESCRIPTION
**Pull request by karlosvalverde**
Current 404 page template has been improved, by adjusting the alignment
of the elements and making a better use of their previous Bootstrap
syntax, in order to have a more readable template.

Task 2409545

@Cocographique 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/website_templates.xml [hunk 1]
```diff
@@ -2120,16 +2120,19 @@
...
-                <a role="button" class="btn btn-primary js_disable_on_click post_link" t-attf-href="/website/add/#{ path }#{ from_template and '?template=%s' % from_template }">Create Page</a>
+        <div class="bg-100 pt-1">
+            <div class="container o_not_editable">
+                <div class="alert alert-info mt32 d-flex justify-content-between align-items-center">
+                    <p class="m-0 text-primary">This page does not exist, but you can create it as you are editor of this site.</p>
+                    <a role="button" class="btn btn-primary js_disable_on_click post_link" t-attf-href="/website/add/#{ path }#{ from_template and '?template=%s' % from_template }">Create Page</a>
+                </div>
+                <div class="text-center"><i class="fa fa-info-circle"></i> Edit the content below this line to adapt the default <strong>Page not found</strong> page.</div>
             </div>
-            <div class="text-center text-muted">Edit the content below this line to adapt the default "page not found" page.</div>
```
Comment (Cocographique) [inline addons/website/views/website_templates.xml:2130]: What do you think?

Or something else?
Screenshot 2020-12-02 at 11 58 29

File: addons/http_routing/views/http_routing_template.xml [hunk 2]
```diff
@@ -122,7 +122,9 @@
                                 <div class="col-12">
                                     <p class="text-center pt24">Maybe you were looking for one of these <b>popular pages?</b></p>
```
Comment (Cocographique) [inline addons/http_routing/views/http_routing_template.xml:123]: As `.pt24` is on a paragraph, it's not editable.
Move this class to the column (which should be `.col-lg-12,` not `.col-12`).

**Comment by JKE-be** [comment]
This page has been already refacored for v14.
Fp ask to remove explicitly all other link that Homepage.

We add even introduce a system to add /shop when it was installed, and he ask to remove all this code and link !

So don't add it again please ;)

//cc @rdeodoo  lol

**Comment by JKE-be** [comment]
ps2: since nobody will edit this page by default, if we can't avoid to add link that point to existing page... 

We removed about us page in v14 !

File: addons/website/views/website_templates.xml (hunk 1 above)
Comment (karlosvalverde) [reply addons/website/views/website_templates.xml:2130]: What about this?

image

File: addons/http_routing/views/http_routing_template.xml [hunk 3]
```diff
@@ -122,7 +122,9 @@
...
                                     <ul class="list-inline text-center">
-                                        <li class="list-inline-item mb-2"><a href="/" class="btn btn-secondary">Home</a></li>
+                                        <li class="list-inline-item mb-2"><a href="/" class="btn btn-primary">Home</a></li>
+                                        <li class="list-inline-item mb-2"><a href="/page/about-us" class="btn btn-primary">About us</a></li>
```
[the remark this answers is not in the document]
Comment (Cocographique) [reply addons/http_routing/views/http_routing_template.xml:126]: So, as explained by JKE, just keep the home button:
`<li class="list-inline-item mb-2"><a href="/" class="btn btn-primary">Home</a></li>`

File: addons/http_routing/views/http_routing_template.xml (hunk 2 above)
Comment (karlosvalverde) [reply addons/http_routing/views/http_routing_template.xml:123]: What about removing this `pt24` and setting the padding of the `hr` above (div in L120) to `p-5` instead?

Comment (Cocographique) [reply addons/http_routing/views/http_routing_template.xml:123]: Yes! And the `.s_hr` snippet should be used here. 
Like in the second slide of the carousel snippet:
[https://github.com/odoo/odoo/blob/def8eb4b51a80587f2b4a17da82ceb3abb7f017c/addons/website/views/snippets/s_carousel.xml#L35](https://github.com/odoo/odoo/blob/def8eb4b51a80587f2b4a17da82ceb3abb7f017c/addons/website/views/snippets/s_carousel.xml#L35)

File: addons/website/views/website_templates.xml (hunk 1 above)
Comment (Cocographique) [reply addons/website/views/website_templates.xml:2130]: Nevermind.
Maybe, just put back the `.text-muted` to make a visual difference with the editable content.

Comment (karlosvalverde) [reply addons/website/views/website_templates.xml:2130]: Hmm not so sure about this. Maybe it works better without the `alert alert-info` classes.

image

image

Comment (Cocographique) [reply addons/website/views/website_templates.xml:2130]: Sorry, it was not clear 😁 
Yes, remove the alert box and put back the muted style.

**Comment by karlosvalverde** [comment]
Alright, I've just force-pushed the changes I've made related to this review.

@Cocographique @JKE-be

**Comment by JKE-be** [comment]
Hey @karlosvalverde,

Please retarget thie 'improvement' in master please.

**Pull request closed without merging**